ihmc-high-level-behaviors/src/libgdx/java/us/ihmc/rdx/ui/graphics/RDXRawImagePointCloudRenderer.java

```diff
@@ -109,8 +109,9 @@ public void updateMesh(RawImage depthImage, RawImage colorImage)
 
       // Update color image uniforms
       colorIntrinsics = colorImage.getIntrinsicsCopy();
-      depthPose.getReferenceFrame().getTransformToDesiredFrame(depthToColorTransform, colorImage.getPose().getReferenceFrame());
-      LibGDXTools.toLibGDX(depthToColorTransform, tempColorTransform, libGDXColorTransform);
+      colorToDepthTransform.setAndInvert(colorImage.getPose());
+      colorToDepthTransform.multiply(depthPose);
+      LibGDXTools.toLibGDX(colorToDepthTransform, tempColorTransform, libGDXColorTransform);
```

Comment:
Fixes a transform issue. Seems to me that we're using a color to depth transform, not depth to color.